charts/vllm/templates/deployment.yaml

```diff
@@ -105,17 +105,23 @@ spec:
             - mountPath: {{ .Values.readManyPVC.mountPath }}
               name: {{ include "vllm.readManyPVCName" . }}
             {{- end }}
+          {{- with .Values.volumeMounts }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
       volumes:
-         - name: dshm
-           emptyDir:
-              medium: Memory
-         {{- if .Values.readManyPVC.enabled }}
-         - name: {{ include "vllm.readManyPVCName" . }}
-           persistentVolumeClaim:
-             claimName: {{ include "vllm.readManyPVCName" . }}
-             readOnly: true
-         {{- end }}
-      {{- with .Values.nodeSelector }}
+        - name: dshm
+          emptyDir:
+             medium: Memory
+        {{- if .Values.readManyPVC.enabled }}
+        - name: {{ include "vllm.readManyPVCName" . }}
+          persistentVolumeClaim:
+            claimName: {{ include "vllm.readManyPVCName" . }}
+            readOnly: true
+        {{- end }}
+        {{- with .Values.volumes }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with .Values.nodeSelector }}
```

Comment:
the identation seems wrong on:
```
{{- with .Values.nodeSelector }}
```

Reply:
you shouldn't have to modify lin 124 at all

Reply:
Updated, good catch